code/modules/vore/fluffstuff/custom_clothes_vr.dm

```diff
@@ -806,6 +806,13 @@
 	rolled_sleeves = 0
 	starting_accessories = list(/obj/item/clothing/accessory/black)
 
+//Kisukegema:Kisuke `the nerd` Gema
+/obj/item/clothing/glasses/sunglasses/omnihud/kamina
+	name = "Kamina glasses"
+	icon_state = "kamina"
+	desc = "ROW ROW, FIGHT THE POWER."
+	see_invisible = SEE_INVISIBLE_LEVEL_TWO //DO THE IMPOSSIBLE, SEE THE INVISIBLE
```

Comment:
Yeah, can cut that out. Mostly put it in as joke. Can't remember the last time we HAD an invisible thing.